_data/team-members.yml

```diff
@@ -39,13 +39,13 @@ members:
     twitter: jvican
     photo: /resources/img/jorge.png
     currentlyWorkingOn: true
-    contact_for: [scalaPlatformProcess, sip, spores, buildTools]
+    contact_for: [scalaPlatformProcess, sip, spores, buildTools, scalacProfiling]
```

Comment:
You're not working on spores though... Maybe remove it?

Reply:
Or wait, no, it's OK. It doesn't show in the percentages, doh.

Reply:
Updated 👍

Reply:
I spoke too soon; I think you need spores to remain in your project list so that you show up on the project page as having worked on it. ...You might have to put it back 🤔